5.6/Dockerfile.rhel7

```diff
@@ -0,0 +1,61 @@
+FROM openshift3/sti-base
```

Comment:
if that's the case we'll probably cherry pick this and deliver it in another PR.

To explain the base image "nightmare":  when we build these upstream repositories, we use a different set of docker registries than what OSBS is building and pushing to.

That means the sti-base upstream repo rhel image (openshift/base-rhel7), when built, gets pushed to an internal registry that we use for QE purposes and to ensure we can do a CI/CD flow.  The other images (eg this one) need to be based on that same openshift/base-rhel7 image so they can also be built+tested in that environment, continuously.

We then have the downstream dist-git repos which are completely internal and built by OSBS.  The output of OSBS builds are the openshift3/<whatever> images.  That means that when we pull code from this repo down into the dist-git repo, there are some changes that have to be made, including changing what the FROM image is, because the downstream(dist-git) repo for sti-base gets pushed to a different location (openshift3/sti-base?  i'm actually not 100% certain of the name offhand) when built by OSBS.

i hope that helps.  the main thing to keep in mind is that this repo and the dist-git repo do not have identical content.